nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/SplitText.java

```diff
@@ -288,7 +290,7 @@ public void process(InputStream in) throws IOException {
         int fragmentIndex = 1; // set to 1 to preserve the existing behavior *only*. Perhaps should be deprecated to follow the 0,1,2... scheme
         String fragmentId = UUID.randomUUID().toString();
 
-        if (computedSplitsInfo.size() == 0) {
+        if ((computedSplitsInfo.size() == 0) && (headerFlowFile != null)) {
```

Comment:
I don't know why it tries to create `splitFlowFile` even though `computedSplitsInfo` is empty. I tested this situation with ver 1.0, but SplitText didn't emit anything if the incoming flow file only contains header.
Is there any reason to do this? @olegz